scripts/wattlab_parity_diff.py

```
@@ -394,294 +399,302 @@ def compare_topology(oracle_dir: Path) -> list[dict]:
+            sev = "noise"
+            delta = None
+            match_status += 1
             match_hours += 1
-    if screening_status or screening_hours:
+        elif status_ok and not hours_ok:
+            sev = "blocker"
+            delta = r_h - o_h
+            match_status += 1
+        else:
+            sev = "blocker"
+            delta = "status mismatch" if not hours_ok else "status mismatch"
```

Ruff (0.16.1)

[warning] Useless `if`-`else` condition

(RUF034)